### PR TITLE
Fix W3C errors

### PR DESCRIPTION
**Pull request by l33d4n**
- Change links 'alt' to 'title'
- Remove Stray end tag 'span'
- Add 'alt' to hero images
- Move JS script inside 'body'

Full list: https://validator.w3.org/nu/?showsource=yes&doc=https%3A%2F%2Fbtcpayserver.org%2F

image

### CODE REVIEW

File: source/src/html/menu-tmpl.html [hunk 1]
```diff
@@ -102,19 +101,19 @@
                             d="M4 0c-.28 0-.5.22-.5.5s.22.5.5.5.5-.22.5-.5-.22-.5-.5-.5zm-2.5 1c-.28 0-.5.22-.5.5s.22.5.5.5.5-.22.5-.5-.22-.5-.5-.5zm5 0c-.28 0-.5.22-.5.5s.22.5.5.5.5-.22.5-.5-.22-.5-.5-.5zm-2.5 1c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-3.5 1.5c-.28 0-.5.22-.5.5s.22.5.5.5.5-.22.5-.5-.22-.5-.5-.5zm7 0c-.28 0-.5.22-.5.5s.22.5.5.5.5-.22.5-.5-.22-.5-.5-.5zm-6 2.5c-.28 0-.5.22-.5.5s.22.5.5.5.5-.22.5-.5-.22-.5-.5-.5zm5 0c-.28 0-.5.22-.5.5s.22.5.5.5.5-.22.5-.5-.22-.5-.5-.5zm-2.5 1c-.28 0-.5.22-.5.5s.22.5.5.5.5-.22.5-.5-.22-.5-.5-.5z">
                         </path>
                     </svg>
-                    </span></a>
+				</a>
```
Comment (dennisreimann) [inline source/src/html/menu-tmpl.html]: Let's fix the indentation to consistenly use 2 spaces while we're at it :)

File: source/src/html/menu-tmpl.html [hunk 2]
```diff
@@ -146,7 +144,7 @@
                         d="M4 0c-.28 0-.5.22-.5.5s.22.5.5.5.5-.22.5-.5-.22-.5-.5-.5zm-2.5 1c-.28 0-.5.22-.5.5s.22.5.5.5.5-.22.5-.5-.22-.5-.5-.5zm5 0c-.28 0-.5.22-.5.5s.22.5.5.5.5-.22.5-.5-.22-.5-.5-.5zm-2.5 1c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-3.5 1.5c-.28 0-.5.22-.5.5s.22.5.5.5.5-.22.5-.5-.22-.5-.5-.5zm7 0c-.28 0-.5.22-.5.5s.22.5.5.5.5-.22.5-.5-.22-.5-.5-.5zm-6 2.5c-.28 0-.5.22-.5.5s.22.5.5.5.5-.22.5-.5-.22-.5-.5-.5zm5 0c-.28 0-.5.22-.5.5s.22.5.5.5.5-.22.5-.5-.22-.5-.5-.5zm-2.5 1c-.28 0-.5.22-.5.5s.22.5.5.5.5-.22.5-.5-.22-.5-.5-.5z">
                     </path>
                 </svg>
-                </span></a>
+			</a>
```
Comment (dennisreimann) [inline source/src/html/menu-tmpl.html]: Indentation :)

**Review by dennisreimann — APPROVED** [review]
Good job :)

File: source/src/html/menu-tmpl.html (hunk 2 above)
Comment (l33d4n) [reply source/src/html/menu-tmpl.html]: Fixed :)

**Review by dennisreimann — APPROVED** [review]

**Pull request merged**